src/lib.rs

```diff
@@ -60,52 +60,83 @@
 //! }
 //! ```
 
+#![no_std]
```

Comment:
wasn't it problematic in other crates when this was always set even if `feature="std"` is set?